server/apps/publish/subscribers.py

```diff
@@ -18,6 +20,9 @@
 
 logger = logging.getLogger(__name__)
 
+subscriber_types = ['broadcast', 'digital', 'wire']
+SUBSCRIBER_TYPES = namedtuple('SUBSCRIBER_TYPES', ['BROADCAST', 'DIGITAL', 'WIRE'])(*subscriber_types)
```

Comment:
If that's the case then we need  a new type for which the takes package is sent as well as the takes themselves. One example of such a subscriber is PublicApi.